client/pages/claim.tsx

```diff
@@ -9,67 +9,94 @@ import Claim from "components/claim/Claim";
-  const [claimOpen, setClaimOpen] = useState(claimIsOpen());
-
-  useEffect(() => {
-    // check every second so component re-renders when counter reaches 0
-    const claimOpensTimer = setInterval(() => {
-      setClaimOpenTsPassed(claimOpenTimestampPassed());
-      setClaimOpen(claimIsOpen());
-    }, 1000);
-
-    return () => {
-      clearInterval(claimOpensTimer);
-    };
-  }, []);
 
   return (
-    <div className="space-y-6">
+    <Wrapper narrow>
       <Seo title="Claim OGV" />
-      {!claimOpenTsPassed && (
-        <Wrapper narrow>
-          <HoldingPage />
-        </Wrapper>
-      )}
-      {claimOpen && (
-        <>
-          <Wrapper narrow>
-            <StepTracker currentStep={currentStep} steps={steps} />
-          </Wrapper>
-          <Wrapper narrow>
-            {currentStep == 0 && (
-              <Eligibility handleNextStep={handleNextStep} />
-            )}
-            {currentStep == 1 && <Education handleNextStep={handleNextStep} />}
-            {currentStep == 2 && <Claim handlePrevStep={handlePrevStep} />}
-          </Wrapper>
-        </>
-      )}
-    </div>
+      <PageTitle>Claim</PageTitle>
+      <Card margin>
+        <div className="bg-accent text-white -mt-10 -mx-6 p-10 md:-mx-10 text-center">
+          <h2 className="text-[1.75rem] line-height-[2.125rem]">
+            The claim period has now expired
+          </h2>
+          <div className='mt-3 text-sm font-bold'>
+            {`All unclaimed OGV was burned after 90 days. You can still buy OGV and stake it for up to four years. Current OGV stakers are earning a vAPY of ${stakingApy.toFixed(2)}%.`}
+          </div>
+          <div className='mt-6'>
+            <Link
+              className={"w-[15.5rem] btn rounded-full normal-case space-x-2 btn-lg h-[3.25rem] min-h-[3.25rem] bg-white border-none"}
+              href="/stake"
+            >
+              {"Create a new stake"}
+            </Link>
+          </div>
+        </div>
+        <div className='-mb-10 -mx-6 p-10 md:-mx-10 text-center'>
+          <div>
+            <Link
+              className={"w-[15.5rem] btn rounded-full normal-case space-x-2 btn-lg h-[3.25rem] min-h-[3.25rem] btn-primary border-none"}
+              href="https://blog.originprotocol.com/tokenomics-retroactive-rewards-and-prelaunch-liquidity-mining-campaign-for-ogv-1b20b8ab41c8"
+              type="external"
+              newWindow
+            >
+              {"Read about the airdrop"}
+            </Link>
+          </div>
+          <div className='mt-2'>
+            <Link
+              className={"w-[15.5rem] btn rounded-full normal-case space-x-2 btn-lg h-[3.25rem] min-h-[3.25rem] btn-primary border-none"}
+              href="https://www.ousd.com/burn"
+              type="external"
+              newWindow
+            >
+              {"See the OGV burn"}
+            </Link>
+          </div>
+        </div>
+      </Card>
+    </Wrapper>
   );
 };
 
 export default ClaimPage;
+
```

Comment:
Should this commented out section be removed?

Reply:
whoops, removed